example/metadata/lib/src/generated/metadata.pb.dart

```diff
@@ -0,0 +1,109 @@
+///
```

Comment:
Very funky. Not sure why it was done this way, or why the 'generated code' comment isn't a dartdoc comment...

There's an open issue on including the comments from the .proto file in the generated .dart files, so I'm planning to address it at that point. Comments were originally left out because they caused larger load times in Dartium, I believe.